sdk/identity/Azure.Identity/tests/SharedTokenCacheCredentialTests.cs

```diff
@@ -164,11 +164,7 @@ public async Task MultipleAccountsNoTenantIdOrUsername()
 
             var ex = Assert.ThrowsAsync<CredentialUnavailableException>(async () => await credential.GetTokenAsync(new TokenRequestContext(MockScopes.Default)));
 
-            Assert.True(ex.Message.StartsWith("Multiple accounts were found in the cache."));
-
-            Assert.True(ex.Message.Contains($"username: fakeuser@fakedomain.com tenantId: {fakeuserTenantId}"));
-
-            Assert.True(ex.Message.Contains($"username: madeupuser@madeupdomain.com tenantId: {madeupuserTenantId}"));
+            Assert.AreEqual(ex.Message, "SharedTokenCacheCredential authentication unavailable, multiple accounts were found in the cache use username and tenant id to disambiguate.");
```

Comment:
Sentence structure. Let's review all of them to ensure we have proper punctuation.